Backend/app.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const recipe = require('./model/model');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+require('dotenv').config();
+
+const app = express();
+
+mongoose
+	.connect(process.env.mongo, {
+		useNewUrlParser: true,
+		useUnifiedTopology: true,
+	})
+	.then(() => console.log('Connexion à MongoDB réussie !'))
+	.catch(() => console.log('Connexion à MongoDB échouée !'));
+
+mongoose.set('useCreateIndex', true);// avoid deprecated message, non essential
+app.use(cors());
+app.use(express.static('public'));
+app.use('/', express.static('public'));
+app.use(bodyParser.json());
+
+app.get('/api', (req, res) => {
+	recipe
+		.find({ recommend: false })
+		.then((recipe) => {
+			return res.send(recipe);
+		})
+		.catch((error) => {
+			return res.json({ error });
```

Comment:
Don't forget to include the error code: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status (same for every route actually, feel free to add the status code everywhere)
```suggestion
			return res.status(500).json({ error });
```